test/dashboard-e2e/tests/test-logs.spec.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-restricted-syntax */
+import {test} from '@playwright/test';
+
+import {ApiHelpers} from '../api/api-helpers';
+import {TestDataHandler} from '../data-handlers/test-data-handlers';
+import {MainPage} from '../pages/MainPage';
+import {TestExecutionsPage} from '../pages/TestExecutionsPage';
+
+const apiHelpers = new ApiHelpers(process.env.API_URL, process.env.CLOUD_CONTEXT, process.env.BEARER_TOKEN);
+
+test(`Run test logs`, async ({page}) => {
```

Comment:
Logs are something that can/should be checked in `run-test.spec.ts` - separate test isn't needed.
There are 3 cases there (`['cypress-git-created', 'k6-git-created', 'postman-git-created']`). All of them are currently aborted not to waste resources, and because cypress execution can be slow. In these case I think it's better to execute k6 or postman test, and check the logs there.